image-index.md

```diff
@@ -55,7 +55,10 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 
     - **`os.version`** *string*
 
-        This OPTIONAL property specifies the operating system version, for example `10.0.10586`.
+        This OPTIONAL property specifies the version of the operating system included in the layers specified by the manifest.
+        Implementations MAY refuse to use manifests of which `os.version` is not known to work with the host OS version.
+        When OS is Windows, image indexes SHOULD use, and implementations SHOULD understand quad-notation values of [`System.Version` properties](system-version), `Major.Minor.Build.Revision`. e.g. `10.0.14393.1066`.
```

Comment:
“OS is Windows” → “`os` is `windows`”, and similarly in the following line.  And I think we want to be implementation-defined for all OSes, with the [Windows matching so fuzzy][1].  @jstarks seemed to be [ok with that approach][2].

[1]: https://github.com/opencontainers/image-spec/pull/632#issuecomment-295018429
[2]: https://github.com/opencontainers/image-spec/pull/632#issuecomment-295498056